docs/tinlake/overview/pricing_rwa.md

```diff
@@ -1 +1,78 @@
+---
+id: pricing-rwa
+order: 4
+title: "Tinlake Pricing and Valuation Series — Part 1"
+category: 1. Overview
+redirect_from:
+- /tinlake/further-information/pricing-rwa/
+---
 
+## How To Price Real-World Assets
+*This post is co-written by Dennis Wellmann, Product Manager at Centrifuge and Howard Krieger, Founder of Residual Token Inc. Howard is an experienced valuation professional with nearly 20 years of experience and specializes in the valuation of fixed income securities and complex financial instruments. He is a Managing Director for a top 10 U.S. accounting firm, a former Big-4 Senior Manager responsible for a variety of structured finance valuation assignments, and a leading pricing expert in the crypto community.*
+
+## Introduction
+Centrifuge’s Tinlake is an asset-backed smart contract lending platform built to responsibly bridge real-world assets, such as invoices, mortgages or streaming royalties to the DeFi ecosystem. This post is the first in a multiple part series about how valuation and pricing of **real-world assets** work within the Tinlake ecosystem. Collectively, this series will form our knowledge base on real-world asset pricing and will be used to educate the DeFi community as we bridge non-native crypto assets into the decentralized finance ecosystem.
+This post starts with the **fundamentals of pricing real-world assets.** It will focus on one particular asset — an invoice — and explain which factors are usually considered in pricing this asset and how these factors may be turned into an interest rate. Invoices are particularly interesting collateral for Tinlake and DeFi because they have historically low default rates and a short maturity of 30–90 days.
+
+## Financing an Invoice
+For our purposes, an invoice is a bill sent from one entity (supplier) to another (buyer) in connection with the delivery of goods or services. Key data points on an invoice usually are:
+1. Invoice amount
+2. Counterparty name and address
+3. Issuance date
+4. Due date
+5. Line items (e.g. type and quantity of goods)
+6. Payment terms and conditions
+
+Suppliers around the world typically have to wait for 30–90 days to get paid for an invoice. Instead of waiting several months until the invoice is due, the supplier has an alternative: find a way to convert the invoice into capital quickly by financing it through a capital provider, e.g. an Asset Originator or a factor.
+
+The parties involved in issuing and financing an invoice through Tinlake usually are the following:
+
+![](./pricing-rwa-1.png)
```

Comment:
Can we use a proper table in markdown instead? That shouldn't have to be an image (medium doesn't support tables - our docs do).